protected/components/Utils.php

```diff
@@ -63,4 +63,14 @@ public static function getFirstWords($string, $numberOfWords) {
 
         return $data;
     }
+
+    /**
+     * Check for chars specific for html in string
+     * @param $string
+     * @return bool
+     */
+    public static function containsSpecificHtmlSymbols($string){
+        return preg_match("/<[^<]+>/",$string,$m) != 0;
```

Comment:
Due to our coding standards, we require spaces after ",": preg_match("blabla", $string, $m);